hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/constants/breadcrumbs.constants.tsx

```diff
@@ -26,5 +26,6 @@ export const breadcrumbNameMap: IBreadcrumbNameMap = {
   '/Pipelines': 'Pipelines',
   '/MissingContainers': 'Missing Containers',
   '/Insights': 'Insights',
-  '/DiskUsage': 'Disk Usage'
+  '/DiskUsage': 'Disk Usage', 
+  '/HeatMap': 'HeatMap'
```

Comment:
IMO should be lower case 'm':

```suggestion
  '/Heatmap': 'Heatmap'
```

but if it is too much effort to change it we file another jira